draft-twi-reference-architecture.md

```diff
@@ -49,22 +49,44 @@ informative:
       org: Confidential Computing Consortium Trustworthy Workload Identity SIG
 
 --- abstract
-This Internet-Draft covers a gap analysis performed by the Confidential Computing Consortium on WIMSE identifying areas where the current WIMSE architecture should be extended to accommodate workloads running in confidential computing environments. It outlines high-level requirements for the these extensions and describes a series of use cases.
+This Internet-Draft covers a gap analysis performed by the Confidential Computing Consortium on WIMSE identifying areas where the current WIMSE Architecture should be extended to accommodate Workloads running in Confidential Computing environments. It outlines high-level requirements for the these extensions and describes a series of use cases.
```

Comment:
Capitalizing terms that are not introduced yet in the abstract is not a huge problem, but is typically started after the terminology section. I am not objecting. At this point the "issue is basically "immaterial".

Reply:
I am going to opt for consistency of capitalization of certain words throughout the document. We can always change that later.